webapp_settings/production.py

```diff
@@ -0,0 +1,47 @@
+# Only set this to True in development environments
+DEBUG = False
+SHOW_BANNER = False
+CAN_EDIT_ELECTIONS: False
+
+# Set this to a long random string and keep it secret
+# This is a handy tool:
+# https://www.miniwebtool.com/django-secret-key-generator/
+SECRET_KEY = "{{ production_django_secret_key }}"
+MEDIA_ROOT = "{{ django_media_root }}"
+STATICFILES_DIRS = ()
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': '{{project_name}}',
+        'USER': '{{project_name}}',
+        'PASSWORD': '',
+    },
+}
+
+# A tuple of tuples containing (Full name, email address)
+ADMINS = (
+    ('YNR Prod Developers', 'developers+ynr-prod@democracyclub.org.uk')
+)
+
+# **** Other settings that might be useful to change locally
+
+ALLOWED_HOSTS = ['*']
```

Comment:
Do you definitely want `ALLOWED_HOSTS = ['*']` here rather than a whitelist?

Reply:
Maybe not, but it's the current setting on prod and stage so I'd rather not change it along with…everything else.

Reply:
baby steps ;)